articles/People/Global_Moderation_Team/en.md

```diff
@@ -0,0 +1,52 @@
+
+Additionally, they are responsible for regulating the upkeep of any issues that may arise in the game, and are responsible for handling reports and appeals; and matters that relate to the game and its community as a whole.
+
+## Responsibilities
+
+The Global Moderation Team is responsible for the upkeep of the game, and ensuring that various aspects of the game run smoothly. It is split into two categories, which can be conveniently split per member of the team, depending on what aspect of the team they are assigned to.
+
+All members of the Global Moderation Team are responsible for the following:
+
+- **Content**: Ensuring that all content in the game is properly moderated, and that any issues that may arise are dealt with accordingly.
+- **Communication**: Communicating with the other members of the Global Moderation Team, as well as other teams, and the community at a whole.
+
+In terms of the community, they are responsible for acting as a go-to for the community, and should be able to answer any questions that the community may have, or at least be able to point them in the right direction.
+
+Additionally, all members of the Global Moderation Team are expected to take part in reviews surrounding the [Content usage guidelines](/wiki/Rules/Content_usage_guidelines)
+
+### Leadership
+
+Leadership members of the Global Moderation Team are responsible for the leadership of the team, but with an emphasis on being a point of connection between the rest of the team, and the community. Unlike the leadership category of the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team), they can act largely internally, and are not required to be a point of contact for the community; although they are still required to be able to communicate with the community if necessary.
+
+### Forum
+
+Members of the Global Moderation Team that are assigned to this category of the team are responsible for the upkeep of the [community forum](/forums). They are required to be active on the forum, and are responsible for the upkeep and moderation of the forum.
+
+### In-Game
+
+Members of the Global Moderation Team that are assigned to this category of the team are responsible for the upkeep of the game. This includes issues regarding the game, as well as the chat.
+
+## Promotion
+
+Promotion to the Global Moderation Team is determined on a case-by-case basis, ranging from the candidate's ability to communicate with the community, to their ability to keep up with the game and its community. The process of finding candidates is a largely internal process, and can take a while to find the right candidate.
+
+Unlike the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team), it is a generalized role that applies to the entire game, and holds no particular requirements other than the aformentioned requirements.
```

Comment:
```Suggestion
```

Reply:
This doesn't exactly need to be here since those wanting to be in the NAT would need to be a CN anyways. Addition to the GMT as a BN should be evaluated from the NAT standpoint since they share the same concerns but will be at an elevated position to the GMT, imo. 

I share this sentiment that if you are CN and GMT, you might as well be in a structural NAT or not share the responsibilities of evaluation NATs. Can be talked about